src/OidcClient/OidcClient.cs

```diff
@@ -313,13 +313,15 @@ public virtual async Task<UserInfoResult> GetUserInfoAsync(string accessToken, C
         /// <param name="refreshToken">The refresh token.</param>
         /// <param name="backChannelParameters">Back-channel parameters</param>
         /// <param name="cancellationToken">The cancellation token.</param>
+        /// <param name="scope">Space separated list of the requested scopes.</param>
         /// <returns>
         /// A token response.
         /// </returns>
         public virtual async Task<RefreshTokenResult> RefreshTokenAsync(
             string refreshToken, 
             Parameters backChannelParameters = null, 
-            CancellationToken cancellationToken = default)
+            CancellationToken cancellationToken = default,
+            string scope = null)
```

Comment:
Could you make scope the 3rd parameter?

Reply:
It does already...

Reply:
@leastprivilege  this change breaking the functionally from interceptor i.e. RefreshTokenDelegatingHandler when token is refreshed but scope is default is null and pass from options. due to that refresh token always fails.

Reply:
fix could be :

var response = await _oidcClient.RefreshTokenAsync(_refreshToken, null, _oidcClient.Options.Scope, cancellationToken: cancellationToken)